apps/protocol-frontend/src/components/ContributionsTable.tsx

```diff
@@ -39,33 +39,10 @@ import EditContributionForm from './EditContributionForm';
 import { UIContribution } from '@govrn/ui-types';
 import DeleteContributionDialog from './DeleteContributionDialog';
 import { BLOCK_EXPLORER_URLS } from '../utils/constants';
-import { useContributions } from '../contexts/ContributionContext';
 import { GovrnSpinner } from '@govrn/protocol-ui';
 import InfiniteScroll from 'react-infinite-scroll-component';
-import TableEndMessage from './TableEndMessage';
```

Comment:
Do we have another way we want to display this after the results are fully loaded? Did you come across any specific best practices recommended for handling this with React Query? I noticed it wasn't showing up at the bottom of the results

Reply:
I don't think we need to have a message. Typically the end just doesn't load anymore data

Reply:
I agree. It'll be obvious it's done and saves UI real estate